File: CS/PS/복습/progr_lv1/행렬의덧셈.py
def solution(arr1, arr2):
    answer = []
    for x in range(0, len(arr1)):
        lst = []
        for y in range(0, len(arr1[0])):
            lst.append(arr1[x][y]+arr2[x][y])
        answer.append(lst)
    return answer

File: CS/PS/Site/programmers/lv1/행렬의덧셈.js
function solution(arr1, arr2) {
  var answer = [[]];
  answer = arr1;
  const length = arr1.length;
  const colLength = arr1[0].length;

  for(let i = 0; i < length; i++){
    for(let k = 0; k < colLength; k++){
      answer[i][k] = arr1[i][k] + arr2[i][k];
    }
  }
  return answer;
}


console.log(solution([[1,2],[2,3]], [[3,4],[5,6]]));
console.log(solution([[1],[2]], [[3],[4]]));
